src/codeu/chat/client/core/Controller.java

```diff
@@ -112,4 +112,80 @@ public ConversationHeader newConversation(String title, Uuid owner)  {
 
     return response;
   }
+
+  @Override
+  public void newConvoInterest(Uuid userId, Uuid idToSave,
+	int numberOfMessagesInConvo) {
+	try (final Connection connection = source.connect()) {
+
+      Serializers.INTEGER.write(connection.out(), NetworkCode.NEW_CONVO_INTEREST_REQUEST);
```

Comment:
The autoformat adds tabs instead of spaces and was giving me issues when merging last time. Is it okay to just auto format everything i commit? (even other's code)